### ISSUE TITLE
listRecordLinks LWC doesn't work correctly

### ISSUE THREAD

**Issue by plinderman** [trailheadapps/visualforce-to-lwc#130]
The listRecordLinks LWC doesn't open a record. This is easily recreated by simply clicking one of the account names on the component.

To fix it, the navigate component in navigateToRecord.js should be adjusted from:
import { LightningElement, api } from 'lwc';
import { NavigationMixin } from 'lightning/navigation';

export default class NavigateToRecord extends NavigationMixin(
    LightningElement
) {
    @api recordId;
    @api label;

    navigate() {
        this[NavigationMixin.Navigate]({
            type: 'standard__recordPage',
            attributes: {
                recordId: this.recordId,
                actionName: 'view'
            }
        });
    }
}

To:
import { LightningElement, api } from 'lwc';
import { NavigationMixin } from 'lightning/navigation';

export default class NavigateToRecord extends NavigationMixin(
    LightningElement
) {
    @api recordId;
    @api label;

    navigate(e) {
        e.preventDefault();
        this[NavigationMixin.Navigate]({
            type: 'standard__recordPage',
            attributes: {
                recordId: this.recordId,
                actionName: 'view'
            }
        });
    }
}

In summary, without e.preventDefault(), the <a> tag's default navigation takes effect instead of the onclick handler's.

**Comment by albarivas** [comment]
I've just deployed the app and tested and it works in Chrome 88. Which browser are you using? Maybe you configured the recipe differently?

**Comment by paullinder-ctl** [comment]
I'm on Chrome 88. 

I didn't do any additional configuration either. I followed the prescribed steps to deploy this to my newly created Trailhead Playground.

When I click one of the account names on the table:

![image](https://user-images.githubusercontent.com/73713997/108439315-a3ed0a80-7216-11eb-9f79-67e4fb710939.png)

It opens a new browser tab and focuses it:

![image](https://user-images.githubusercontent.com/73713997/108439340-b2d3bd00-7216-11eb-8607-c2076cb2192b.png)

The original browser tab also gets redirected to the same location:
![image](https://user-images.githubusercontent.com/73713997/108439402-d0a12200-7216-11eb-8a31-7faddf4a6dac.png)

**Comment by albarivas** [comment]
Ok, so that's because you added target="_blank". The original recipe opens the account record page on the same location. Makes sense?

**Comment by albarivas** [comment]
You are right! Thank you for highlighting it. Fixed.

**Comment by paullinder-ctl** [comment]
Thank you for looking into it!

I couldn't have implemented this feature in the first place without your helpful example
